### DISCUSSION TITLE
How to dislike a song?

### DISCUSSION THREAD

**Discussion by KucharczykL** [jellyfin/jellyfin-web#4156] [Q&A] [answered]
Hello, the other day  I have noticed that if you go to your music library -> Songs and click on the filter button, there is "Likes" and "Dislikes" filter, in addition to "Favorites". 

My question is: how can I "Like" or "Dislike" a song when the only button is the heart which marks it as "Favorite"?

![image](https://user-images.githubusercontent.com/31072879/200293368-3ab13080-4404-491c-8ec1-2dbb743af3d9.png)

**Comment by nielsvanvelzen** [comment, marked as answer]
It's not possible to like or dislike items anymore. This filter option therefor does nothing.

**Reply by KucharczykL** [reply]
Ah shame, I would have liked (ha!) to be able to dislike a song.
